CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### 🦊 What's Changed 🦊
 
+- When a rollout audience size decreases, then the re-evaluate the enrollment, and unenroll if no longer in the bucket. ([#5687](https://github.com/mozilla/application-services/pull/5687)).
```

Comment:
The way this is written right now is a tad confusing, I'd definitely suggest rewording it a little. You're welcome to use the below suggestion or you can reword it differently, totally up to you

```suggestion
- When a rollout audience size decreases, the enrollment is re-evaluated and the client is unenrolled if no longer in the bucket. ([#5687](https://github.com/mozilla/application-services/pull/5687)).
```